book/controller.rst

```diff
@@ -143,7 +150,7 @@ Mapping a URL to a Controller
 
 The new controller returns a simple HTML page. To actually view this page
 in your browser, you need to create a route, which maps a specific URL path
-to the controller:
+to the controller::
```

Comment:
It's only 2 `::` right before a `php` block (and then we don't need the `.. code-block:: php`